base/linalg/uniformscaling.jl

```diff
@@ -167,6 +167,8 @@ end
 
 inv(J::UniformScaling) = UniformScaling(inv(J.λ))
 norm(J::UniformScaling, p::Real=2) = abs(J.λ)
+det(J::UniformScaling{T}) where {T} = one(T)
```

Comment:
I guess you could check if `\lambda == 1` or else throw, but doesn't seem very useful ?